### PR TITLE
[ACR] Add 202 response to PUT, 204 response to DELETE.

### PR DESCRIPTION
**Pull request by djyou**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

File: arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json [hunk 1]
```diff
@@ -140,6 +143,9 @@
         "responses": {
           "200": {
             "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "204": {
```
Comment (fearthecowboy) [inline arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json]: Just to be clear, you don't want an error thrown in this case.

Currently, if you specify a response code, it indicates that the result is a success condition. You'd want to put a "default" in as a catch-all for error conditions (which `...does not exist in the subscription` sounds a bit like an error code)

Comment (djyou) [inline arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json]: Correct, we don't want an error thrown when deleting a resource that doesn't exist.

Comment (fearthecowboy) [inline arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json]: Ok, I'm fine with that.

**Pull request merged**